yjit/src/backend/arm64/mod.rs

```diff
@@ -971,6 +975,16 @@ impl Assembler
             cb.link_labels();
         }
 
+        #[cfg(feature = "disasm")]
```

Comment:
@k0kubun Alan was saying that you should be able to avoid duplicating any logic for x86/arm by moving the logic into the `compile()` method which is platform-agnostic.

Reply:
ah, I didn't notice it. will fix it that way. thanks

Reply:
fixed in https://github.com/Shopify/ruby/pull/430/commits/4dd74628630c09ff7710904a0e59b9d15a972c4a